azure-pipelines.yml

```
@@ -0,0 +1,106 @@
+        parameters:
+          status: 'failure'
+          webhookUrl: '$(DiscordWebhook)'
+          message: ''
+
+      - task: PublishPipelineArtifact@1
+        displayName: Publish Pipeline Artifact copy
+        inputs:
+          path: '$(System.DefaultWorkingDirectory)'
+          artifactName: 'Artifacts'
+
+- ${{ each appComponents in parameters.appComponents }}:   
```

yamllint

[error] trailing spaces

(trailing-spaces)